IntegerManipulationPractise/Program.cs

```diff
@@ -11,58 +11,36 @@ public class Program
-            Console.WriteLine("How many numbers per list?");
+            Console.WriteLine("How many numbers per list?(less or equal than 10)");
             var elementCount = Convert.ToInt32(Console.ReadLine());
-            var startTime = DateTime.UtcNow;
-            var testList = new List<int>();
-            var skupRandomLista = new List<List<int>>();
-            for (var i =0; i < numOfLists; i++)
+            Console.WriteLine("How many shuffles?");
+            var numOfShuffles = Convert.ToInt32(Console.ReadLine());
+            var generatedSortedSet = new SortedSet<int>();
+            var random = new Random();
+            var totalElements = numOfLists*elementCount;
+            for (int i = 0; i < totalElements; i = generatedSortedSet.Count)
             {
-                //var randomList = new List<int>();
-                //randomList.AddRange(IntegerUtils.GetRandomNumbers(elementCount));
-                //for (var broj = 0; broj < elementCount; broj++)
-                //{
-                //    for (var broj1 = i ; broj1 >= 0 ; broj1--)
-                //    {
-                //        while (testList[broj*i] == randomList[broj])
-                //        {
-                //            randomList.RemoveAt(broj);
-                //            randomList.Insert(broj, IntegerUtils.GetRandomNumbers(1)[0]);
-                //        }
-                //    }
-                //}
-                //testList.AddRange(randomList);
+                generatedSortedSet.Add(random.Next(0, 101));
+            }//do sada radi, izgenerise listu sa numOfLists*elementCount elemenata...
+            var generatedList = generatedSortedSet.ToList();
+            Console.WriteLine("\n");
+            foreach (var element in generatedList)
+            {
+                Console.Write("{0}, ", element);
+            }
+            Console.WriteLine("\n");
 
 
-                var randomList = new List<int>();
-                randomList.AddRange(IntegerUtils.GetRandomNumbers(elementCount));
-                while (randomList.SequenceEqual(testList))  //dva puna dana sam se mucio oko ovoga dok nisam shvatio da .equals i == ne poredi liste clan po clan.........
-                {
-                    randomList.Clear();
-                    randomList.AddRange(IntegerUtils.GetRandomNumbers(elementCount));
-                }
-                testList.Clear();
-                testList.AddRange(randomList);
-                skupRandomLista.Add(randomList);//ICloneable!!!
-                Console.WriteLine("\nUnsorted list No.{0}: ", i+1);
-                foreach (var num in randomList)
-                {
-                    Console.Write(num + " ");
-                }
-                Console.WriteLine("\nSorted List No.{0}: ", i+1);
+            var shuffledList = Shuffle.ShuffleListRandomInsert(generatedList);//ako ne napravim novu listu ovde, nece da mi prikaze shuffle-ovano, ZASTO?
```

Comment:
Hoces da kazes, hteo si da shuffle ovu originalnu listu "in place", bez da pravis novu?